### PR TITLE
Document environment variables options usage

### PR DESCRIPTION
**Pull request by t-chaik**
See https://github.com/google/benchmark/issues/881 and https://github.com/google/benchmark/issues/883.

### CODE REVIEW

**Comment by t-chaik** [comment]
From https://github.com/google/benchmark/pull/883:

> Hm, one last thought - we probably want something about this in docs?
> But that can come in next pr.

Please, et me know what you think @LebedevRI.

File: README.md [hunk 1]
```diff
@@ -282,17 +288,51 @@ too (`-lkstat`).
...
 [Disabling CPU Frequency Scaling](#disabling-cpu-frequency-scaling)
 
+<a name="running-benchmarks" />
+
+### Running Benchmarks
+
+Benchmarks are executed by running the produced binaries. Benchmarks binaries,
+by default, accept options that may be specified either through their command
+line interface or by setting environment variables before execution. For every
+`--option_flag=<value>` CLI swich, a corresponding variable
```
Comment (LebedevRI) [inline README.md]: corresponding *environment* variable

File: README.md [hunk 2]
```diff
@@ -282,17 +288,51 @@ too (`-lkstat`).
...
+`OPTION_FLAG=<value>` exist and is used as default if set (CLI switches always
+ prevails). A complete list of CLI options is available running benchmarks
+ with the `--help` switch.
+
+<a name="running-a-subset-of-benchmarks" />
+
+### Running a Subset of Benchmarks
+
+The `--benchmark_filter=<regex>` option (or `BENCHMARK_FILTER=<regex>`
+variable) can be used to only run the benchmarks that match the
```
Comment (LebedevRI) [inline README.md]: *environment* variable

File: README.md [hunk 3]
```diff
@@ -282,17 +288,51 @@ too (`-lkstat`).
...
+
 <a name="output-formats" />
 
 ### Output Formats
 
 The library supports multiple output formats. Use the
-`--benchmark_format=<console|json|csv>` flag to set the format type. `console`
-is the default format.
+`--benchmark_format=<console|json|csv>` (or set
+`BENCHMARK_FORMAT=<console|json|csv>`) flag to set the format type.
```
Comment (LebedevRI) [inline README.md]: environment variable

File: README.md [hunk 4]
```diff
@@ -282,17 +288,51 @@ too (`-lkstat`).
 
 [Disabling CPU Frequency Scaling](#disabling-cpu-frequency-scaling)
 
+<a name="running-benchmarks" />
+
+### Running Benchmarks
```
Comment (LebedevRI) [inline README.md]: I think it would make sense to keep this after `Output Files`, before `Running a Subset of Benchmarks`

**Comment by LebedevRI** [comment]
Thanks, i missed that it was updated.
I'll merge with small tunings.

**Pull request merged**